spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -128,6 +128,7 @@ object HyperConformance extends StreamTransformerFactory with HyperConformanceAt
       datasetName = conf.getString(datasetNameKey),
       datasetVersion = conf.getInt(datasetVersionKey),
       reportDate = new SimpleDateFormat(ReportDateFormat).format(new Date()),
```

Comment:
```suggestion
      reportDate = new SimpleDateFormat(ReportDateFormat).format(new Date()),  // Still need a report date for mapping table patterns
```
I think the comment was useful there.